### PR TITLE
Bumps version

### PR DESCRIPTION
**Pull request by abea**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "apostrophe-redirects",
-  "version": "2.0.0",
+  "version": "2.1.0",
```
Comment (boutell) [inline package.json]: Update the changelog (end of the README)

Comment (abea) [reply package.json]: good call.

**Comment by boutell** [comment]
you may publish sir

**Comment by abea** [comment]
Published! 💣 all the tabs.

**Pull request merged**